shub/image/build.py

```diff
@@ -67,7 +67,7 @@ def build_cmd(target, version, skip_tests):
     click.echo("The image {} build is completed.".format(image_name))
     # Test the image content after building it
     if not skip_tests:
-        test_mod.test_cmd(target, version)
+        test_cmd.test_cmd(target, version)
```

Comment:
ahaha, ok @jdemaeyer @vshlapakov, I'll fix this :) 2 options: 

1. I can rename it back 
2. import command only

```
from shub.image.test import test_cmd
# ...
test_cmd(target, version)
```
I'd go with the last one